mindmeld/converter/README.md

```diff
@@ -0,0 +1,7 @@
+### Mindmeld Conversion Tool
```

Comment:
Are we going to update this?

Reply:
I believe Sanjna said she will be updating it